### PR TITLE
Enable lint check

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: migration/migration_test.go [hunk 1]
```diff
@@ -405,9 +405,11 @@ func writeToFile(path string, data string) (err error) {
 		return err
 	}
 
-	file.WriteString(data)
+	defer file.Close()
 
-	file.Close()
+	if _, err := file.WriteString(data); err != nil {
+		return err
```
Comment (al1img) [inline migration/migration_test.go:411]: aoserros.Wrap

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -29,3 +29,9 @@ jobs:
```
Comment (al1img) [inline .github/workflows/ci.yaml:29]: Fix commit message: Lint -> lint

File: wsserver/wsserver.go [hunk 3]
```diff
@@ -127,7 +127,9 @@ func (server *Server) Close() {
 		client.close(true)
 	}
 
-	server.httpServer.Shutdown(context.Background())
+	if err := server.httpServer.Shutdown(context.Background()); err != nil {
+		log.Errorf("Can't shutdown server")
```
Comment (al1img) [inline wsserver/wsserver.go:131]: add ': %s', err

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (Eabramov84) [reply .github/workflows/ci.yaml:29]: fixed

File: migration/migration_test.go (hunk 1 above)
Comment (Eabramov84) [reply migration/migration_test.go:411]: added wrap

File: wsclient/wsclient.go [hunk 4]
```diff
@@ -240,7 +243,12 @@ func (client *Client) SendMessage(message interface{}) (err error) {
 
 	log.WithFields(log.Fields{"client": client.name, "message": string(messageJSON)}).Debug("Send message")
 
-	client.connection.SetWriteDeadline(time.Now().Add(client.clientParam.WebSocketTimeout))
+	if err := client.connection.SetWriteDeadline(time.Now().Add(client.clientParam.WebSocketTimeout)); err != nil {
+		log.WithFields(log.Fields{"client": client.name}).Debugf("Can't set write deadline timeout: %s", err)
```
Comment (al1img) [inline wsclient/wsclient.go:247]: Add empty line after log.

**Review by al1img — APPROVED** [review]

**Pull request merged**